tests/README.md

```diff
@@ -1,3 +1,137 @@
+# Getting Started
+
+##  What are UI Tests?
```

Comment:
In general, I think we should move this either later section of this document or to README in the tests/ui

Reply:
I moved to the tests/ui